src/Pim/Bundle/ApiBundle/tests/integration/Controller/Product/SuccessLargeAndOrderedListProductIntegration.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Pim\Bundle\ApiBundle\tests\integration\Controller\Product;
+
+use Doctrine\Common\Collections\Collection;
+use Pim\Component\Catalog\Model\ProductInterface;
+use Pim\Component\Catalog\tests\integration\Normalizer\NormalizedProductCleaner;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * We want to test the API is capable of returning an ordered list of 100 items.
+ * ie, twice the size of a cursor page
+ */
+class SuccessLargeAndOrderedListProductIntegration extends AbstractProductTestCase
+{
+    /** @var Collection */
+    private $products;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if (1 === self::$count) {
+
+            $identifiers = [];
+            for ($i = 0; $i < $this->getListSize(); $i++) {
+                $identifiers[] = 'sku-' . str_pad($i, 4, '0', STR_PAD_LEFT);
+            }
+
+            foreach ($identifiers as $identifier) {
+                $product = $this->createProduct($identifier, []);
+                $this->products[$product->getId()] = $product;
+            }
+            // the API will return products sorted by MySQL ID, and that's what we expect
```

Comment:
you can say that it's an alphabetical order (it's not a natural way of sorting ids). But it's understandable like that too